daemon/daemon_unix.go

```diff
@@ -556,8 +556,8 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 	return warnings, nil
 }
 
-// platformReload updates configuration with platform specific options
-func (daemon *Daemon) platformReload(config *Config) map[string]string {
+// reloadPlatform updates configuration with platform specific options
+func (daemon *Daemon) reloadPlatform(config *Config, attributes map[string]string) {
```

Comment:
What you mentioned is what I am concerned with as well. I hope that we could add a specific comments as your suggested. Updated this PR. PTAL